json-rpc/src/stream_rpc/subscriptions.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
nit: Lobbying to rename this to `subscription_types.rs` or something more differentiated from `subscription.rs`